README.md

```diff
@@ -8,71 +8,87 @@ transfers made easy<br><br>
 
 [![CI](https://github.com/astronomer/apache-airflow-provider-transfers/actions/workflows/ci-uto.yaml/badge.svg)](https://github.com/astronomer/apache-airflow-provider-transfers)
 
-The **Universal Transfer Operator** simplifies how users transfer data from a source to a destination using [Apache Airflow](https://airflow.apache.org/). It offers a consistent agnostic interface, improving the users' experience so they do not need to use explicitly specific providers or operators.
+The **UniversalTransferOperator** simplifies how you transfer data from a source to a destination using [Apache Airflow](https://airflow.apache.org/). Its agnostic interface eliminates the need to use specific providers or operators.
 
-At the moment, it supports transferring data between [file locations](https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/src/universal_transfer_operator/constants.py#L26-L32) and [databases](https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/src/universal_transfer_operator/constants.py#L72-L74) (in both directions) and cross-database transfers.
+At the moment, it supports transferring data between [file locations](https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/src/universal_transfer_operator/constants.py#L26-L32) and [databases](https://github.com/astronomer/apache-airflow-provider-transfers/blob/main/src/universal_transfer_operator/constants.py#L72-L74), as well as cross-database transfers.
 
 This project is maintained by [Astronomer](https://astronomer.io).
 
 ## Installation
 
-```
+```sh
 pip install apache-airflow-provider-transfers
 ```
 
-
 ## Example DAGs
 
-Checkout the [example_dags](./example_dags) folder for examples of how the UniversalTransfeOperator can be used.
+See the [example_dags](./example_dags) folder for examples of how you can use the UniversalTransfeOperator.
 
-
-## How Universal Transfer Operator Works
+## How the UniversalTransferOperator works
 
 ![Approach](./docs/images/approach.png)
 
-With Universal Transfer Operator, users can perform data transfers using the following transfer modes:
+The purpose of the UniversalTransferOperator is to move data from a source dataset to a destination dataset. Your datasets can be defined as `Files` or as `Tables`. 
+
+Instead of using different operators for each of your transfers, the UniversalTransferOperator supports three universal transfer types:
 
-1. Non-native
-2. Native
-3. Third-party
+- Non-native transfers
+- Native transfers
+- Third-party transfers
 
+### Non-native transfers
```

Comment:
Is it too late to rename these transfer types / is this the common term for this type of transfer? This was unintuitive to me, because I considered a worker-driven transfer to be native, but from the perspective that it's _native to Airflow_. I'd prefer if we called these:

- Worker transfer
- Dataset transfer
- Third-party transfer

Reply:
@jwitz I think we can still rename these transfers, as we have not done a major release yet. 
IMO,
1. Worker transfer - Makes sense to me as well
2. For Native transfer - We can maybe use - direct / peer to peer
3. Third-party transfer - This is perfect as it is. 

Also, maybe we can lose the `transfer` from names since we already call them `transfer modes`. 
WDYT?

Reply:
IMO - we should rename them to the below to be more easily understood by users:

1."local"
2."optimized"
3."third-party"

Reply:
IMO - we should rename them to the below to be more easily understood by users:

1."local"
2."optimized"
3."third-party"

Reply:
I'm not sure "optimized" will help users understand exactly what's going on. Optimized can mean many things, and some might consider third party to be the "optimized" solution for their use case.

I like "peer-to-peer" or "direct" @sunank200 !

@utkarsharma2 I think we should still keep "Transfer", because it helps us communicate what these modes are in documentation. If you want to remove "transfer" at the code level in terms of how people specify these modes in the operator parameters, I think that's fine.

Reply:
@jwitz Sure we can keep the transfer in docs, but we can remove it from the code. @sunank200 @phanikumv  WDYT?